tests/build/build_templates.go

```diff
@@ -327,7 +327,9 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 				Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("failed to parse SBOM from show-sbom task output from %s/%s PipelineRun", pr.GetNamespace(), pr.GetName()))
 				Expect(sbom.BomFormat).ToNot(BeEmpty())
 				Expect(sbom.SpecVersion).ToNot(BeEmpty())
-				Expect(sbom.Components).ToNot(BeEmpty())
+				if !strings.Contains(gitUrl, "from-scratch") {
+					Expect(sbom.Components).ToNot(BeEmpty())
+				}
```

Comment:
just curious, why do we expect non-zero sbom components when the giturl is `https://github.com/konflux-qe-bd/docker-file-from-scratch`?

Reply:
from scratch means it is [empty image](https://hub.docker.com/_/scratch) in the dockerfile and for the components list as per [the spec](https://cyclonedx.org/schema/bom-1.5.schema.json) of cyclonedx format of SBOM, it is `A list of software and hardware components` , since the base image is empty so there is no software component, it would be empty

Reply:
Thanks for the explanation!